test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -273,7 +277,7 @@
   (let [settings (assoc (testutils/ca-sandbox! cadir)
                         :allow-duplicate-certs true
                         :autosign false)
-        request {:authorization {:name "authname"} :remote-addr "1.1.1.1" }]
+        request {:authorization {:name "authname"} :remote-addr "1.1.1.1"}]
```

Comment:
Cursive made this, and the paren changes below, because the indentation didn't match the parens.